schemas/apiDefinition.swagger.schema.json

```diff
@@ -2278,6 +2345,19 @@
         }
       ]
     },
+    "x-ms-enum": {
+      "description": "NOTE: This extension is ignored by the Power Platform. The schema is only provide here for convenience.",
```

Comment:
The same, owners can specify a lot of different custom extensions in swaggers, unlikely we will add all of them. Moreover, it can give a false confidence these extensions are used by Power Platform.

Reply:
When the description has this NOTE in the description, it's pretty clear it's not supported.
Again, the goal is to be able to look in the Problems window and identify potential typos, missing properties, etc.